src/conn/mod.rs

```diff
@@ -50,11 +50,18 @@ use crate::{
 
 use self::routines::Routine;
 
+use regex::bytes::Regex;
+
 pub mod binlog_stream;
 pub mod pool;
 pub mod routines;
 pub mod stmt_cache;
 
+lazy_static::lazy_static! {
+    static ref FIXED_MARIADB_VERSION_RE: Regex =
+        Regex::new(r"^(?:5.5.5-)?(\d{1,2})\.(\d{1,2})\.(\d{1,3})-MariaDB").unwrap();
```

Comment:
If you're a regex guru, please think about this regex 🙏.

Before:
```rust
Regex::new(r"^5.5.5-(\d{1,2})\.(\d{1,2})\.(\d{1,3})-MariaDB")
```

After:
```rust
Regex::new(r"^(?:5.5.5-)?(\d{1,2})\.(\d{1,2})\.(\d{1,3})-MariaDB")
```

All I did was make `5.5.5-` an optional group. Lemme know if you see ways this could go wrong.